requirements/base.in

```diff
@@ -0,0 +1,6 @@
+pylint==2.6.0
+pylint-django==2.3.0
+pylint-celery==0.3
+six>=1.10.0,<2.0.0
+click>=6.0
+click-log==0.3.2
```

Comment:
Any idea what constraints are needed at all, and which are needed here vs in a constraints.txt? See https://github.com/edx/edx-cookiecutters/blob/3079f93e1f72a7d5339e2340fd4a13e49af2c4eb/cookiecutter-django-app/%7B%7Bcookiecutter.repo_name%7D%7D/requirements/base.in#L2
Note: minimum constraints are fine here.  It is the pinning and maximum constraints that I am wondering about.

Reply:
I tried to be backward compatible, so I just copy-pasted the content from setup.cfg. In the future, requirements should not be pinned in the base.in requirement file.

To be honest, I don't understand the point of having a contraints.txt when we can simply have a base.in. I guess that constraints.txt are for temporary contraints?

Reply:
Yes. The pins and upper constraints can all get properly documented for when and how they can be removed.

Maybe just add a TODO comment to clarify the next step?

Reply:
:ok_hand: